File: java/src/main/java/itp/itp1_3/ITP1_3_C.java
package itp.itp1_3;

import java.io.PrintWriter;
import java.util.Scanner;

public class ITP1_3_C {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    PrintWriter pw = new PrintWriter(System.out);
    while (true) {
      int x = sc.nextInt();
      int y = sc.nextInt();
      if (x == 0 && y == 0)  break;
      pw.append(x > y ? y + " " + x : x + " " + y);
      pw.append("\n");
    }
    pw.flush();
    pw.close();
    sc.close();
  }
}

File: kotlin/src/itp/itp1_3/ITP1_3_C.kt
package itp.itp1_3

import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    while (true) {
        val x = sc.nextInt()
        val y = sc.nextInt()
        if (x == 0 && y == 0) break
        println("${Math.min(x,y)} ${Math.max(x,y)}")
    }
}